src/common/utils.py

```diff
@@ -10,17 +10,23 @@
 previous_state = {}
 
 
-def show(state, device_name, debug_mode=False):
+def create_message(state, device_name):
+    state_ext = {
+        "device_name": device_name,
+    }
+    state.update(state_ext)
```

Comment:
I don't think we should change the input parameter `state` inside this method. Making a copy, updating the copy and returning it is probably the way to go. This behavior can be separated from the `create_message` function (wouldn't expect a method with that name to also have the side effect of changing the input parameter).
Sorry for the multiple change requests on this part!